tests/test_cli.py

```diff
@@ -15,7 +15,7 @@ def test_cli():
         main()
```

Comment:
we can add a basic test for the API endpoint like [this](https://github.com/Lightning-AI/litserve/blob/9f0c054d1c1eeb90fd882fb6988404ad0f4f207e/tests/test_simple.py#L38). 

```py
def test_simple():
    server = LitServer(SimpleLitAPI(), accelerator="cpu", devices=1, timeout=30)

    with TestClient(server.app) as client:
        response = client.post("/predict", json={"prompt": "Some random prompt"})
        assert response.json() == "Expected model output"
```

Reply:
Sounds good, I will use a small offline model that doesn't have to be downloaded, because the CI is already suffering haha